recipes/lazy_loader/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lazy_loader" %}
+{% set version = "0.1rc2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lazy_loader-{{ version }}.tar.gz
+  sha256: 1130fe8ddf64e7bfbeea185e14a08575b55a4680ed1585482eaeb796faca242b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit-core
```

```suggestion
    - flit-core >=3.3,<4
```

https://github.com/scientific-python/lazy_loader/blob/d132b4d60a976f6d027580605d7a5fc84b50b06e/pyproject.toml#L2